indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
Ah I see your point. So doing BytesWritableInputRowParser as something in druid-indexing-hadoop would make sense to me- in this case we'll ask users that want to read BytesWritable SequenceFiles to change two things: the inputFormat and the type of the parser.

I think it makes sense in core rather than an extension, because BytesWritable is a core hadoop type. If we wanted to add official support for something like Avro at some point then that would make sense to me as an extension.